website/src/pages/dictionary/index.js

```diff
@@ -144,30 +145,29 @@ function DataDictionary() {
         const { primaryId = false, core = false, dependsOn = false } = meta;
         const restrictions = get(field, 'restrictions', false);
         if (primaryId) {
-          acc.validDataTiers.add(TAG_TYPES.id);
+          acc.validDataTiers.push(TAG_TYPES.id);
         }
 
         if (!!restrictions) {
-          acc.validDataAttributes.add(TAG_TYPES.required);
+          acc.validDataAttributes.push(TAG_TYPES.required);
         }
 
         if (dependsOn) {
-          acc.validDataAttributes.add(TAG_TYPES.dependency);
+          acc.validDataAttributes.push(TAG_TYPES.dependency);
         }
 
         if (core) {
-          acc.validDataTiers.add(TAG_TYPES.core);
+          acc.validDataTiers.push(TAG_TYPES.core);
         }
 
         if (!core && !primaryId) {
-          acc.validDataTiers.add(TAG_TYPES.extended);
+          acc.validDataTiers.push(TAG_TYPES.extended);
         }
         return acc;
       },
-      { validDataTiers: new Set(), validDataAttributes: new Set() },
+      { validDataTiers: [], validDataAttributes: [] },
     );
-
-    setFilters({ tiers: [...validDataTiers], attributes: [...validDataAttributes] });
+    setFilters({ tiers: uniq(validDataTiers), attributes: uniq(validDataAttributes) });
```

Comment:
outside of scope for this, but `filters` just seems to be a transformation of `schemaFields`, doesn't need to be a state eh?

Reply:
@hlminh2000 yup! nice catch. will update in another PR